src/signals/incident/components/form/MapSelectors/Asset/Selector_v2_removeafterfinishepic5440/WfsLayer/utils/map-data-to-selectable-feature.ts

```diff
@@ -50,6 +50,21 @@ export const mapDataToSelectableFeature = (
           type: typeValue,
         }
       })
+    case FeatureTypes.CATERPILLAR:
+      return features.map((feature) => {
+        const { description, typeValue } = featureTypes[0]
+
+        const coordinates = featureToCoordinates(feature?.geometry as Geometrie)
```

Comment:
Shouldn't the type be called `Geometry`?

Reply:
True, but that's not for this PR. Should be a refactor since it will touch many files